specification/ai/ModelClient/main.tsp

```diff
@@ -10,7 +10,7 @@ using Azure.Core;
 
 #suppress "@azure-tools/typespec-autorest/unsupported-http-auth-scheme"
 @useAuth(
-  ApiKeyAuth<ApiKeyLocation.header, "api-key"> | BearerAuth | OAuth2Auth<[
```

Comment:
Has the service team finally fixed this?

Reply:
From offline conversation, generally seems like nothing has changed yet.